### PR TITLE
Create ui-c8y-1018-503-87-adjust-global-role-title-for-testing-save-button-context-in-sso.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by amio

Ticket: [MTM-59237](https://cumulocity.atlassian.net/browse/MTM-59237)
Original PR: [6333](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6333)

### CODE REVIEW

File: content/change-logs/platform-services/ui-c8y-1018-503-87-adjust-global-role-title-for-testing-save-button-context-in-sso.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: adjust global role title for testing save button context in sso spec (#6333)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1018-503-87-adjust-global-role-title-for-testing-save-button-context-in-sso.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Adjust global role title in SSO spec for testing save button
```

File: content/change-logs/platform-services/ui-c8y-1018-503-87-adjust-global-role-title-for-testing-save-button-context-in-sso.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-0UgqXH1Ys
+    label: Administration
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-59237
+version: 1018.503.87
+---
+adjust global role title for testing save button context in sso spec (#6333)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1018-503-87-adjust-global-role-title-for-testing-save-button-context-in-sso.md:17]: The Claude AI suggested to use the following body:
```suggestion
To improve the testing of the save button in the SSO specification, the title of a global role has been adjusted. This change modifies the global role title used specifically in the SSO spec's test cases. The impact of this change is limited to the SSO specification's test environment and does not affect any production systems or user-facing functionality.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by kalavathig1** [comment]
[MTM-59237](https://cumulocity.atlassian.net/browse/MTM-59237) is a test fix, we dont need release notes for that. Hence closed

**Pull request closed without merging**